index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Eureka Caffe</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico" />
+    <link rel="stylesheet" href="style.css" />
+    <link href="nouislider.css" rel="stylesheet" />
+    <script src="nouislider.js"></script>
+    <script rel="javascript" src="index.js"></script>
+  </head>
+
+  <header>
+    <div class="navbar">
+      <div class="left-logo">
+        <a href="#">
+          <img src="images/logo.png" alt="Eureka Coffe logo" />
+        </a>
+        <a href="#">
+          <p>
+            Eureka <br />
+            Coffe
+          </p>
+        </a>
+      </div>
+      <div class="right-menu">
+        <a href="#">Home </a>
+        <i class="bi bi-circle-fill"></i>
+        <a href="#">About </a>
+        <i class="bi bi-circle-fill"></i>
+        <a href="#">Services </a>
+        <i class="bi bi-circle-fill"></i>
+        <a href="#">Blog </a>
+        <i class="bi bi-circle-fill"></i>
+        <a href="#">Contacts </a>
+      </div>
+    </div>
+  </header>
+
+  <body>
+    <div class="welcome-container">
+      <div class="welcome-text">
+        <h2>Eureka Caffe</h2>
+        <p>
+          Satisfy your cravings with our hassle-free online ordering experience.
+          From delectable dishes to your doorstep, we serve up convenience and
+          mouthwatering meals that will leave you craving for more.
+        </p>
+      </div>
+    </div>
+
+    <div class="bar">
+      <div class="categories">
+        <div class="selection">
+          <span onclick="toggle(this, 'arrow1')">
```

Comment:
I think this method name is a bit too generic, since it's internal implementation should toggle the dropdown menu.